tests/test_updater.py

```diff
@@ -478,6 +482,9 @@ def test_idle(self, updater, caplog):
         with caplog.at_level(logging.INFO):
             updater.idle()
 
+        if caplog.records[-1].getMessage().startswith('Error while getting Updates: Conflict'):
```

Comment:
wait, what? Will need to double check, but I IIRC we have 1 bot token per test suite, so this should never happen?!

Reply:
Yes, we have 3.6-3.9 on 3 OSes = 4×3 = 12 suits and 12 test bots … But this can happen when running tests for different PRs in parallel, probably? We should at least add a comment for that, IG

Reply:
> But this can happen when running tests for different PRs in parallel, probably? 

It happens roughly once a week iirc, so yeah

> We should at least add a comment for that, IG

ok will include in next commit once I find out why the first two tests fail at times